tests/MongoDB.Driver.Core.Tests/Specifications/connection-string/TestRunner.cs

```diff
@@ -39,6 +35,8 @@ public class TestRunner
         public void RunTestDefinition(JsonDrivenTestCase testCase)
         {
             var definition = testCase.Test;
+            JsonDrivenHelper.EnsureAllFieldsAreValid(definition, "valid", "options", "hosts", "auth", "description", "uri", "warning");
+
```

Comment:
We've been slowly renaming our `TestRunner` classes to more descriptive names.

Since we're working on this class let's rename it to `ConnectionStringTestRunner`.